cyipopt/ipopt_wrapper.py

```diff
@@ -5,7 +5,7 @@
 from .scipy_interface import IpoptProblemWrapper as CyipoptIpoptProblemWrapper
 from .scipy_interface import convert_to_bytes as cyipopt_convert_to_bytes
 from .scipy_interface import get_bounds as cyipopt_get_bounds
-from .scipy_interface import get_constraint_bounds as cyipopt_get_constraint_bounds
+from .scipy_interface import get_constraint_bounds_and_dimensions as cyipopt_get_constraint_bounds
```

Comment:
What's the reasoning for renaming the function and then importing under an alias? Should we just leave the function name as is? Or if the function name needs to chain for clarity reasons but must remain for compatibility, should we support both and deprecate the old name?

Reply:
Good question. But I'm not quite sure, maybe @moorepants can enlighten us?  Btw: I just noticed it should be `get_constraint_bounds` since I split the function into two smaller ones in one of the previous commits.